target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -12,6 +12,12 @@ board_config_update
 board=$(board_name)
 
 case "$board" in
+askey,rt4230w-rev6)
+	ucidef_add_switch "switch0" \
+		"2:lan" "3:lan" "4:lan" "5:lan" "6@eth1" "1:wan" "0@eth0"
+	ucidef_set_interface_macaddr "lan" "$(mtd_get_mac_binary "0:ART" 0x6)"
+	ucidef_set_interface_macaddr "wan" "$(mtd_get_mac_binary "0:ART" 0x0)"
```

Comment:
MAC addresses in device-tree are preferred, as it keeps the configuration portable across devices. Can you move them over there?

Reply:
Moved it